content/docs/12-enterprise-version/00-on-prem-system-requirements.md

```diff
@@ -51,6 +51,21 @@ The sections below describe the standard requirements and highlight specific req
 **Note**: The exact values for the kubernetes-region and kubernetes-zone tags can be different from the ones described in the example above, as long as these are unique.
 <br />
 
+<InfoBox>
+
+### Naming conventions for vSphere Region and Zone Tags
```

Comment:
This feels like prerequisite

Reply:
yes added as an infobox under prerequisites